### ISSUE TITLE
nav-primary li a:hover, a:focus and a:active have the same colour (#b3002d)

### ISSUE THREAD

**Issue by rudifa** [maban/christmas-recipes#1]
I suggest giving a different colour to a:active, to distinguish it from a:hover, perhaps along these lines:

```
.nav-primary li a:hover,
.nav-primary li a:focus,
.nav-primary .active:hover,
.nav-primary .active:focus {
    background-color: #b3002d;
    color: #fff;
    }

.nav-primary li a:active,
.nav-primary .active,
.nav-primary .active:active {
    background-color: #b300ad;
    color: #fff;
    }
```

Of course, this is just a minor side issue in your excellent tutorial.

rudifa

**Comment by maban** [comment]
Thanks for suggesting this! I've implemented this, tweaked the colour a little to fit in with the theme.

**Comment by rudifa** [comment]
Thank you for taking up my suggestion.

Rudi

On Mon, Feb 17, 2014 at 6:37 PM, Anna Debenham notifications@github.comwrote:

> Closed #1 https://github.com/maban/christmas-recipes/issues/1.
> 
> ## 
> 
> Reply to this email directly or view it on GitHubhttps://github.com/maban/christmas-recipes/issues/1
> .

## 

http://about.me/rudi.farkas
